biojava-genome/src/test/java/org/biojava/nbio/genome/TestChromosomeMappingTools.java

```diff
@@ -0,0 +1,61 @@
+package org.biojava.nbio.genome;
+
+import org.biojava.nbio.genome.util.ChromosomeMappingTools;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static junitx.framework.ComparableAssert.assertEquals;
+
+/**
+ * Created by Yana Valasatava on 8/14/17.
```

Comment:
Could you use the `@author` tag here and also write a minimal explanation of what the test does?